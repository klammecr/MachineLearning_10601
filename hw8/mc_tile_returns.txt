-2.000000000000000000e+02
-1.640000000000000000e+02
-1.780000000000000000e+02
-1.880000000000000000e+02
-1.490000000000000000e+02
-1.090000000000000000e+02
-1.500000000000000000e+02
-1.080000000000000000e+02
-1.060000000000000000e+02
-1.060000000000000000e+02
-1.060000000000000000e+02
-1.050000000000000000e+02
-1.060000000000000000e+02
-1.060000000000000000e+02
-1.880000000000000000e+02
-1.030000000000000000e+02
-1.050000000000000000e+02
-1.740000000000000000e+02
-1.480000000000000000e+02
-1.450000000000000000e+02
-1.440000000000000000e+02
-1.090000000000000000e+02
-1.380000000000000000e+02
-1.260000000000000000e+02
-1.380000000000000000e+02
