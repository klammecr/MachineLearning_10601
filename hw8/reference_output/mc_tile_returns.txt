-2.000000000000000000e+02
-1.650000000000000000e+02
-1.840000000000000000e+02
-2.000000000000000000e+02
-1.440000000000000000e+02
-1.500000000000000000e+02
-1.520000000000000000e+02
-1.540000000000000000e+02
-1.460000000000000000e+02
-1.070000000000000000e+02
-1.060000000000000000e+02
-1.060000000000000000e+02
-1.060000000000000000e+02
-1.080000000000000000e+02
-1.250000000000000000e+02
-1.400000000000000000e+02
-1.450000000000000000e+02
-1.420000000000000000e+02
-1.760000000000000000e+02
-1.050000000000000000e+02
-1.550000000000000000e+02
-1.380000000000000000e+02
-1.380000000000000000e+02
-1.390000000000000000e+02
-1.390000000000000000e+02
